sdk/identity/identity-broker/samples/v1-beta/javascript/README.md

```diff
@@ -0,0 +1,67 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-active-directory
+  - WAM
+  - MSA support
+urlFragment: identity-broker-javascript-beta
+---
+
+# Azure Identity Brokered Auth Plugin client library samples for JavaScript (Beta)
+
+These sample programs show how to use the JavaScript client libraries for Azure Identity Brokered Auth Plugin in some common scenarios.
+
+| **File Name**     | **Description**       |
+| ----------------- | --------------------- |
+| [index.js][index] | Demonstrates usage of |
```

Comment:
The description in the 2nd column needs to be completed. Demonstrates the usage of what?

Reply:
done